WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentsModalButtonViewModel.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-struct CardPresentPaymentsModalButtonViewModel: Identifiable {
+struct CardPresentPaymentsModalButtonViewModel: Identifiable, Equatable {
```

Comment:
I think this case doesn't need the `Equatable` conformance?

```suggestion
struct CardPresentPaymentsModalButtonViewModel: Identifiable {
```

Reply:
Good observation. Would make sense that `Identifiable` is also `Equatable`. I'll check 👍

Reply:
Yes, not needed 👍

Reply:
I think `Hashable` is `Equatable` (it inheirits from it, I believe)

`Identifiable` just means you know that this a specific instance, it doesn't let you determine whether one instance equals another.

Similar to pointer comparison vs equality in C-based languages.